.gitlab/binary_build/system_probe.yml

```diff
@@ -10,6 +10,10 @@
     # TODO: remove this once we switch to k8s runners, they won't have this problem
     - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -depth  # -delete implies -depth
     - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -delete || true  # Allow failure, we can't remove parent folders of datadog-agent
+    # XXX: system-probe images does not run `entrypoint.sh` which runs `source /root/.bashrc`.
+    # Since the path of Go binary is added into PATH in the bashrc, we need to run it at some point.
+    # This is a workaround until I understand why we don't wanna run `entrypoint.sh` for system-probe. 
+    - source /root/.bashrc
```

Comment:
I am a bit surprised, I'd expect the Go binary to be in the path because of that line: https://github.com/DataDog/datadog-agent-buildimages/blob/main/system-probe_x64/Dockerfile#L68-L69
Would you have an example of the failure you get?

Reply:
In buildimages we now download 2 toolchains (Go upstream and Microsoft Go) and the selection is done in `.bashrc` (https://github.com/DataDog/datadog-agent-buildimages/pull/654/files#diff-317dca1ac5a787324dd61e610372375a639415675c46c9a063e380aafc41ea8fR37-R44)